### ISSUE TITLE
Simple "Peer Review" metadata field

### ISSUE THREAD

**Issue by jgmorse** [mlibrary/heliotrope#1533]
This ticket becomes superseded by #516 if it becomes unblocked. 

- [ ] Add a Peer Review metadata field for Monographs/Assets, with a controlled vocabulary of 'yes|no'.
- [ ] Make it non-required; need to support a null state if we don't know. Don't display a label if undefined.
- [ ] Add the column to the submission log / metadata template.
- [ ] The template field should be required for UMPRE workflows.

**Comment by jgmorse** [comment]
Needs to be in place for EBC launch in January 2019. (Add new milestone?)
